### ISSUE TITLE
Animation of population spiking

### ISSUE THREAD

**Issue by rossant** [kwikteam/phy#281]
We can create a GUI for making animations (and videos) of population spiking with VisPy. For example, replay spike waveforms in the waveform view as a function of time, with a custom speed, and a time position slider.

Cluster color could depend on various things such as firing rate, principal frequency (for cells with periodic firing), etc. We could also use place cells for the position.

@nsteinme I'm sure you'll have other ideas :)

**Comment by nsteinme** [comment]
Yes, this is a great idea. Here's a way you can imagine it: the waveform view is to the right, and as the movie plays the waveform that occurs appears there and quickly fades. You only show the unmasked channels, and color each neuron uniquely. Then, streaming out from the waveform view going to the left, is the raster developing over time. Each neuron's raster spikes are (optionally) plotted at the depth of that neuron on the probe. So you'd see a yellow spike near the top of the probe, and a yellow tick mark appears and starts scrolling away from time zero. 

Then, another critical feature for this, you should be able to have an arbitrary number of continuous traces (or other event rasters) also scrolling along with the neural data. So while you watch these rasters, you also see the trace of pupil diameter, whisking, running speed, stimulus onset times, LFPs, etc. 

Here is a movie I made of something fairly similar: https://www.dropbox.com/s/zbtyun34f38n7o5/rasterWaveMovie.mp4?dl=0

**Comment by rossant** [comment]
Population-spiking animations are outside phy’s curation scope.
